src/NHibernate.Test/TestCase.cs

```diff
@@ -48,6 +49,8 @@ protected virtual string MappingsAssembly
 			get { return "NHibernate.DomainModel"; }
 		}
 
+		protected SchemaExport SchemaExport => _schemaExport ?? (_schemaExport = new SchemaExport(cfg));
```

Comment:
```
NHSpecificTest\NH2195\SQLiteMultiCriteriaTest.cs(35,18): warning CS0108: 'SQLiteMultiCriteriaTest.SchemaExport(Configuration)' hides inherited member 'TestCase.SchemaExport'. Use the new keyword if hiding was intended.
```

Release build will fail. Maybe is it time to enable warning as error in the test project debug config too.